.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-  ".": "2.9.6"
+  ".": "2.9.6-rc.1"
```

Comment:
Should this still be 2.9.6?

Reply:
yes it should be 2.9.6, not sure about the `-rc.1` yet though. the `Release-As: 3.0.0-rc.1` comment in the commit body was not being detected as it is supposed to be. This is probably a bug in `release-please`, but given time constraints I'm opting to run a new release pipeline with a versioning strategy of `always-bump-major`.